### PR TITLE
Update creation of function pointer in fastcomp

### PR DESCRIPTION
**Pull request by Mintyboi**
After #10741, we are still getting non-unique function pointer indexes in fastcomp because we are doing a matching the on the stubs instead of the wasm function wrapper.

In this change, we get the actual function wrapper and pass it into addFunction() so that addFunction can use this to compare with the function wrapper gotten from the wasm table.

### CODE REVIEW

File: emscripten.py [hunk 1]
```diff
@@ -1559,14 +1559,19 @@ def check(extern):
       fullname = "fp$" + extern
       key = '%sModule["%s"]' % (side, fullname)
       asm_setup += '''\
-    var %s = function() {
-      if (!%s) { %s
-        var fid = addFunction(%sModule["%s"] || %s, "%s");
-        %s = fid;
+    var %(fullname)s = function() {
+        if (!%(key)s) { %(assetMsg)s
```
Comment (kripken) [inline emscripten.py:1563]: indentation doesn't look right here, should be 2 spaces less I think

**Comment by Mintyboi** [comment]
> Normally I'd insist on a testcase but as this is fastcomp-only code, that seems excessive. 
> But please fix the indentation issue. Thanks!

Sure, I'll fix the indentation and take a look at the CI failures.

**Comment by Mintyboi** [comment]
@kripken I've fixed the indentation but I was not able to reproduce the test failures locally, do you have any idea? It is because I'm on a windows machine?
e.g.
```
Test suites:
['test_core']
Running test_core: (1 tests)
(checking sanity from test runner)
shared:INFO: (Emscripten: Running sanity checks)
test_dylink_funcpointers_wrapper (test_core.wasm0) ... flip
ok

----------------------------------------------------------------------
Ran 1 test in 41.706s

OK
```

**Comment by sbc100** [comment]
Was this perhaps fixed already by #10912?

**Comment by kripken** [comment]
It's strange it just happens on CI and not locally for you @Mintyboi . I'm not sure why that could happen.

I suggest updating this to latest master, as perhaps there was breakage on master at the time you branched?

@sbc100 I think this is a separate issue from that other PR. This avoids using a JS wrapper around a wasm export.

**Comment by Mintyboi** [comment]
Given that fastcomp has been deprecated, this issue should no longer be relevant.
Closing this.

**Pull request closed without merging**